docs/user-docs/latest/content-manager/writing-content.md

```diff
@@ -30,6 +30,10 @@ To write or edit content:
 | Media       | 1. Click the media area. <br> 2. Choose an asset from the [Media Library](/user-docs/latest/media-library/introduction-to-media-library.md) or from a [folder](/user-docs/latest/media-library/organizing-assets-with-folders.md) if you created some, or click the **Add more assets** button to add a new file to the Media Library. <br><br> 💡 It is possible to drag and drop the chosen file in the media area.                                                                                                                                   |
 | JSON        | Write your content, in JSON format, in the code textbox.                                                                                                                                                                                                                                                                                                                  |
 | UID         | Write a unique identifier in the textbox. A "Regenerate" button, displayed on the right of the box, allows to automatically generate a UID based on the content-type name.                                                                                                                                                                                                |
+::: note
```

Comment:
This might just be an Americanization but "filling up" is mostly associated with 3 dimensions in 2 dimensions I think "filling out" is more common.